cachito/web/static/api_v1.yaml

```
@@ -187,14 +187,23 @@ paths:
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  items:
-                    type: array
-                    items:
-                      $ref: "#/components/schemas/Request"
-                  meta:
-                    $ref: "#/components/schemas/Pagination"
+                anyOf:
+                  - type: object
+                    properties:
+                      items:
+                        type: array
+                        items:
+                          $ref: "#/components/schemas/Request"
```

Would it work if we `anyOf` just the items? (might make it clearer that the rest of the response is the same)
```suggestion
                      items:
                        anyOf:
                          - type: array
                            items:
                              $ref: "#/components/schemas/Request"
                          - ...
```

---

I'll give it a try :+1:

---

Thanks for mentioning this. It appears to work just fine and is more succinct :smile: 
I made one additional change: anyOf --> oneOf. I think that makes sense after this change, since the array will be of all either Request or VerboseRequest